core/src/main/java/org/infinispan/metadata/impl/PrivateMetadata.java

```diff
@@ -21,11 +22,13 @@
    /**
     * A cached empty {@link PrivateMetadata}.
     */
-   private static final PrivateMetadata EMPTY = new PrivateMetadata();
+   private static final PrivateMetadata EMPTY = new PrivateMetadata(null);
 
-   //The following fields will be added in later PRs:
-   //IracVersion => used by IRAC (async xsite new algorithm)
-   //SimpleClusteredVersion => used by Optimistic Transactions
+   private final IracMetadata iracMetadata;
```

Comment:
We can make this:

```java
@ProtoField(number = 1)
final IracMetadata iracMetadata
```

That way the marshalling info is always at the top of the class and easier to read.